_data/fellows.yml

```diff
@@ -1,3 +1,18 @@
+gabrielleBeaudry:
+  name: Gabrielle Beaudry
+  info: 'Master of Science candidate in Geographic Information Science for Development and Environment at Clark University'
+  year: 2018
+  image: gabrielleBeaudry.jpg
+xiaoWu:
+  name: Xiao Wu
+  info: 'Master of Science candidate in Geographic Information Science for Development and Environment at Clark University'
+  year: 2018
+  image: xiaoWu.jpg
+daleWatt:
+  name: Dale Watt
+  info: 'Master of Science candidate in Geographic Information Science for Development and Environment at Clark University'
```

Comment:
exact same info as Xiao and Gab - looks like it also matches Sarah Gates. This could be right for Dale, not sure.

Reply:
Updated.